components/bl-pagination/src/index.js

```diff
@@ -54,7 +55,7 @@ export default function Pagination({ component, eventHandlers }) {
 
         { isFirstPageButtonVisible && (
           <FirstPageButton
-            onGoFirst={ onGoFirst }
+            onGoFirst={ onGoFirst.hasLogic ? onGoFirst : component.goFirstPage }
```

Comment:
We skip current page handling if there is user handler? Why?

Reply:
If the user will have some logic in for example On Go First he needs to use the action Go to First Page. But when his logic is empty we use this action by "default"